meta-mender-core/recipes-mender/mender-client/mender-client_git.inc

```diff
@@ -84,10 +84,15 @@ def mender_license(branch):
                    "md5": "3e7426c258f60f9876ae3746f54c49d4",
                    "license": "Apache-2.0 & BSD-2-Clause & BSD-3-Clause & ISC & MIT & OLDAP-2.8",
         }
+    elif branch == "2.4.x":
+        return {
+                   "md5": "5649992d13a6fda40abfac7730a62b07",
+                   "license": "Apache-2.0 & BSD-2-Clause & BSD-3-Clause & ISC & MIT & OLDAP-2.8 & OpenSSL License",
+        }
     else:
         return {
                    "md5": "5649992d13a6fda40abfac7730a62b07",
-                   "license": "Apache-2.0 & BSD-2-Clause & BSD-3-Clause & ISC & MIT & OLDAP-2.8",
+                   "license": "Apache-2.0 & BSD-2-Clause & BSD-3-Clause & ISC & MIT & OLDAP-2.8 & OpenSSL License",
```

Comment:
I believe it's just `OpenSSL`. It maps directly to [this file](https://github.com/mendersoftware/poky/blob/master/meta/files/common-licenses/OpenSSL).

Reply:
Thanks, I will change it. I thought about it, but followed the upstream reference [here](https://www.openssl.org/source/license-openssl-ssleay.txt). (Plus I had no idea that poky had this)